projects/common/src/lib/elements/manage/manage.component.html

```diff
@@ -106,20 +106,25 @@
           </ng-container>
 
           <ng-container *ngIf="MaxDevicesReached">
-            <p class="paragraph-text">
+            <p *ngIf="DevicesConfig.MaxDevicesCount < 50" class="paragraph-text">

```

Comment:
We should not be hard coding to 50 here.  Other licenses could be created that allow more than 50 as the max, which would break this.  You should get that EnterprisesDeviceCount somewhere dynamic, we need to hit the drawing board on this before releasing something static at 50, and somehow get it dynamically.